environments/conda/environment.yml

```diff
@@ -6,23 +6,25 @@ channels:
     - conda-forge
     - bioconda
 dependencies:
-    # - pytorch-gpu=1.13.1 # conda-forge < pytorch + nvidia
-    # - cudatoolkit=11.8 # conda-forge < pytorch + nvidia
-    # - nccl # conda-forge < pytorch + nvidia
-    - python=3.10
-    - pytorch
-    - pytorch-cuda=11.8
-    - torchvision
+    - python=3.11
     - accelerate
     - jupyterlab
     - matplotlib
     - pandas
     - seaborn
     - einops
     - wandb
+    - scipy
     - genomepy
     - sourmash-minimal
+    - bedtools
+    - biopython
+    - pyBigWig
+    - gtfparse
     - pip
     - pip:
-          - gimmemotifs
-          - memory-efficient-attention-pytorch
+        - pytorch
+        - tensorflow[and-cuda]
+        - torchvision
```

Comment:
can we move `pytorch` and `torchvision` back to the conda section to use the pytorch and nvidia channels and add `pytorch-cuda=12.1` above as well in working toward replicating

```sh
conda install pytorch torchvision pytorch-cuda=12.1 -c pytorch -c nvidia
```

which is used in the official [torch Dockerfile](https://github.com/pytorch/pytorch/blob/a0cea517e770fa04f1f33bf6847c6852ca9abc06/Dockerfile#L77)